### ISSUE TITLE
grlc appears to have a memory for broken files

### ISSUE THREAD

**Issue by rlzijdeman** [CLARIAH/grlc#62]
Hi, when making a mistake in an .rq file, grlc won't load your repo, exciting with error:

```
500 : <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN"> <title>500 Internal Server Error</title> <h1>Internal Server Error</h1> <p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p> http://grlc.io/api/rlzijdeman/uncertainty-sparql/spec
```

That's all fine, but it appears that if you fix the mistake, grlc still won't load the repo. This might also be related to #61. When I tried to fix it, at some point I decided to go with a no-fail option: namely by pasting the code from working a .rq file in the .rq file at hand. The code is from [[1]](https://github.com/albertmeronyo/lodapi/blob/master/dbpedia_test.rq)

In the forked repo containing the code from dbpedia_test.rq [[2]](https://github.com/rlzijdeman/uncertainty-sparql) grlc fails [[3]](http://grlc.io/api/rlzijdeman/uncertainty-sparql)

However in a completely new repo [[4]](https://github.com/rlzijdeman/grlc-test/blob/master/test.rq) the pasted code works for grlc [[5]](http://grlc.io/api/rlzijdeman/grlc-test)

I'm experiencing this with other repo's as well, e.g. [[6]](http://grlc.io/api/CLARIAH/wp4-queries-hisco#/)

@albertmeronyo can you think of what would create such a 'memory'?

**Comment by albertmeronyo** [comment]
Hi @rlzijdeman 

I've pushed a commit that should fix #61 , could you try again?

Thanks,
Albert

**Comment by rlzijdeman** [comment]
I guess you mean #62 (this issue)? Cheers! Works like a [charm](http://grlc.io/api/rlzijdeman/uncertainty-sparql), now!
